config/application.rb

```diff
@@ -10,6 +10,7 @@ module HerosTeamApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
+    config.autoload_paths << "#{Rails.root}/app/lib"
```

Comment:
no need to add this,
all directories within `app/` folder already added to autoload paths

remote this or move to a `lib/` directory instead of `app/lib/`

Reply:
I read that lib/ at new versions is not autoload path anymore
app/lib/ it was proposition by Igor)

Reply:
leave it in app/lib

Reply:
yes, `lib/` is not in autoload paths, but `app/lib/` is
it actually doesn't matter if it be in `lib/` or in `app/lib/`
what matters is that file and class names are correct
and of course that directory is included in autoload paths